src/govuk/overrides/_spacing.scss

```diff
@@ -71,13 +71,15 @@ $_spacing-directions: (
 /// @access private
 @mixin _govuk-generate-static-spacing-overrides($property) {
   @each $spacing-point in map-keys($govuk-spacing-points) {
-    .govuk-\!-#{$property}-static-#{$spacing-point} {
+    .govuk-\!-#{$property}-static-#{$spacing-point},
+    .govuk-\!-static-#{$property}-#{$spacing-point} {
       #{$property}: govuk-spacing($spacing-point) !important;
     }
 
     @each $direction in $_spacing-directions {
-
-      .govuk-\!-#{$property}-#{$direction}-static-#{$spacing-point} {
+      /// govuk-#{$property}-#{$direction}-static-#{$spacing-point} is deprecated. Use .govuk-\!-static-#{$property}-#{$direction}-#{$spacing-point} instead.
```

Comment:
Given this applies to the non-directional classes as well, do we either want to add a second comment there or move this either to the top or into the Sassdoc for the function and make it cover both?

Reply:
Good point. I've given it a go, but found it difficult to write anything generic, so it's a bit long... Welcome any comments!

Reply:
Looks great to me 👍🏻